crates/uv/src/lib.rs

```diff
@@ -131,8 +131,21 @@ async fn run(cli: Cli) -> Result<ExitStatus> {
 
     // Parse the external command, if necessary.
     let run_command = if let Commands::Project(command) = &*cli.command {
-        if let ProjectCommand::Run(uv_cli::RunArgs { command, .. }) = &**command {
-            Some(RunCommand::try_from(command)?)
+        if let ProjectCommand::Run(uv_cli::RunArgs {
+            command, script, ..
+        }) = &**command
+        {
+            if *script {
+                let (target, args) = command.split();
+                if let Some(target) = target {
```

Comment:
This was lifted from RunCommand::try_from, I thought it best to not meddle with that API by adding a ```script``` parameter